### ISSUE TITLE
Assets are mising in v0.54.0 release

### ISSUE THREAD

**Issue by fabasoad** [lindell/multi-gitter#518]
**Describe the bug**
[v0.54.0](https://github.com/lindell/multi-gitter/releases/tag/v0.54.0) release does not have assets. So, it is impossible to download and use this version. Even though it is not the latest at the moment it would be great to have assets for consistency, especially that [install.sh](https://github.com/lindell/multi-gitter/blob/master/install.sh) script supports passing specific version into it. Now, command below fails.

```shell
curl -s https://raw.githubusercontent.com/lindell/multi-gitter/master/install.sh | sh -s -- -d "v0.54.0"
```

**To Reproduce**
1. Go to the [v0.54.0](https://github.com/lindell/multi-gitter/releases/tag/v0.54.0) release page.
2. Notice only 2 assets attached:

<img width="241" alt="Screenshot 2024-01-06 at 19 02 40" src="https://github.com/lindell/multi-gitter/assets/2127723/41659b9d-120d-491b-b381-4d204bdad6d8">

**Expected behavior**
Should be 22 assets like in [previous release](https://github.com/lindell/multi-gitter/releases/tag/v0.53.1):

<img width="182" alt="Screenshot 2024-01-06 at 19 02 47" src="https://github.com/lindell/multi-gitter/assets/2127723/4c4f5846-1d75-4c53-b5c6-e3821e20da81">

**Additional context**
None.

**Comment by lindell** [comment]
Is this something that is blocking you or just nice for consistency? 
0.54.0 failed in the CI pipeline the pipeline was fixed and the same content was released as 0.54.1.
It would require **a lot** of time to fix.

**Comment by fabasoad** [comment]
It is not blocking me, just nice to have for consistency. And if someone (for any reason) would like to install `v0.54.0` with `install.sh` script. But if you feel that ROI is not worth it then feel free to close this issue. Thank you for double checking with me before closing it. Appreciate it.
